apollo-core/src/main/java/com/ctrip/framework/foundation/internals/provider/DefaultServerProvider.java

```diff
@@ -16,20 +35,16 @@
 
 public class DefaultServerProvider implements ServerProvider {
   private static final Logger logger = LoggerFactory.getLogger(DefaultServerProvider.class);
-  private static final String SERVER_PROPERTIES_LINUX = "/opt/settings/server.properties";
-  private static final String SERVER_PROPERTIES_WINDOWS = "C:/opt/settings/server.properties";
 
   private String m_env;
   private String m_dc;
 
-  private Properties m_serverProperties = new Properties();
+  private final Properties m_serverProperties = new Properties();
 
   @Override
   public void initialize() {
     try {
-      String path = Utils.isOSWindows() ? SERVER_PROPERTIES_WINDOWS : SERVER_PROPERTIES_LINUX;
-
-      File file = new File(path);
+      File file = ResolvedPaths.SERVER_PROPERTIES.toFile();
```

Comment:
How about we make it a little simpler? 

e.g.
```java
  ...
  File file = new File(getServerPropertiesPath());
  ...

  String getServerPropertiesPath() {
    String serverPropertiesPath = getCustomizedServerPropertiesPath();

    if (Strings.isNullOrEmpty(serverPropertiesPath)) {
      serverPropertiesPath = Utils.isOSWindows() ? SERVER_PROPERTIES_WINDOWS : SERVER_PROPERTIES_LINUX;
    }

    return serverPropertiesPath;
  }

  private String getCustomizedServerPropertiesPath() {
    // 1. Get from System Property
    String serverPropertiesPath = System.getProperty("apollo.serverPropertiesPath");
    if (Strings.isNullOrEmpty(serverPropertiesPath)) {
      // 2. Get from OS environment variable
      serverPropertiesPath = System.getenv("APOLLO_SERVERPROPERTIESPATH");
    }

    return serverPropertiesPath;
  }


```

Reply:
Change to simpler implemention.

How about use `APOLLO_PATH_SERVER_PROPERTIES` to instead of `APOLLO_SERVERPROPERTIESPATH` for better readability?